client/src/components/atoms/style/Tag.scss

```diff
@@ -0,0 +1,8 @@
+.tag {
+    width: -webkit-fit-content;
+    width: -moz-fit-content;
+    width: fit-content;
+    padding: 11px;
+    background-color: #EDEDED;
```

Comment:
could we make `#EDEDED` a global scss variable in `_config.scss`?

Reply:
done!